source/magpie/helper.d

```diff
@@ -37,6 +37,19 @@ template getArgsList(args...)
         alias getArgsList = AliasSeq!();
 }
 
+/// Template to evaluate RowType when column is dropped
```

Comment:
(this may be fixed in a later commit in which case please ignore)

A little bit more description would be good here. something like 

"removes elements in `Types` according to the positions in `pos`. `pos` must be ascending"

Reply:
Addressed in dec86d1